src/backend/distributed/sql/udfs/citus_stat_activity/latest.sql

```diff
@@ -0,0 +1,25 @@
+SET citus.enable_metadata_sync TO OFF;
+
+CREATE TYPE citus.citus_stat_activity_one_node AS (global_pid bigint, is_worker_query boolean, pg_stat_activity pg_stat_activity);
+
+ALTER TYPE citus.citus_stat_activity_one_node SET SCHEMA pg_catalog;
+GRANT ALL ON TYPE pg_catalog.citus_stat_activity_one_node TO PUBLIC;
```

Comment:
I think this is the same currently, as USAGE is the only permission on a type (afaict), but making that explicit seems good from a clarity and security perspective.
```suggestion
GRANT USAGE ON TYPE pg_catalog.citus_stat_activity_one_node TO PUBLIC;
```

Reply:
Great suggestion but I dropped this type too